packages/web-client/src/helpers/space/types.ts

```diff
@@ -35,22 +60,22 @@ export interface PersonalSpaceResource extends SpaceResource {
   __personalSpaceResource?: any
 }
 export const isPersonalSpaceResource = (resource: Resource): resource is PersonalSpaceResource => {
-  return resource?.driveType === 'personal'
+  return (resource as SpaceResource)?.driveType === 'personal'
```

Comment:
Another (maybe cleaner) solution would be to do a type predicate via `isSpaceResource`. But it would require more effort in unit tests because the method checks for `type === 'space'`. That means each time we mock `SpaceResource` we would need to manually set `type` to `space` for this to evaluate to `true`.

Reply:
I like to keep things reasonably simple - keeping it as it is right now is fine with me